### DISCUSSION TITLE
How to configure date and time formats?

### DISCUSSION THREAD

**Discussion by wunderfeyd** [datenstrom/community#47] [Ask a question] [answered]
It would be nice to allow different output date/time formats, either by current user language or by a global setting. This would:
- improve the end user experience with YellowCMS
- reduce the (coding) work to be done by the website owner
- create a better flow in reading content created by the CMS

What do you think?

**Comment by markseuffert** [comment]
Good question, should the date/time format be in the language settings or system settings?

**Comment by wunderfeyd** [comment]
I prefer the language settings. In this case a multilanguage blog/website would deliver the most suitable format automatically. No thinking required, except the basic/standard configuration of YellowCMS which has to be done anyway.

**Comment by markseuffert** [comment, marked as answer]
Thanks, that makes a lot of sense. I added it to the language settings. Here's an example:

```
CoreTimeFormatShort: H:i
CoreTimeFormatMedium: H:i:s
CoreTimeFormatLong: H:i:s T
CoreDateFormatShort: F Y
CoreDateFormatMedium: Y-m-d
CoreDateFormatLong: Y-m-d H:i
```

Please have a look if it's working.

**Comment by wunderfeyd** [comment]
Thank you very much. Seems to work perfectly. Nice work.
